packages/@react-stately/table/src/useTableState.ts

```diff
@@ -51,6 +51,8 @@ export function useTableState<T extends object>(props: TableStateProps<T>): Tabl
     showSelectionCheckboxes: props.showSelectionCheckboxes && selectionState.selectionMode !== 'none',
     selectionMode: selectionState.selectionMode,
     columns: []
+  // Props.children is required here in case new rows are added/removed
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }), [props.children, props.showSelectionCheckboxes, selectionState.selectionMode]);
```

Comment:
This one I'm kinda puzzled about since it doesn't look like context has any reliance on props.children and useCollection already has props.children in it's useMemo, but removing it broke adding rows in the CRUD example

Reply:
I think @devongovett wrote the answer to this at one point in the slack channel but i can't find it. or maybe it was an old comment in a PR. Maybe he'll remember